src/matcha_ml/services/global_config_service.py

```diff
@@ -0,0 +1,139 @@
+"""Global config service for creating and modifying a users global config files."""
+import os
+import uuid
+from typing import Any, Dict, Optional
+
+import yaml
+
+from matcha_ml.errors import MatchaPermissionError
+
+
+class GlobalConfigurationService:
```

Comment:
Nit: I see this class more of as a configuration than service. So, my suggestion would be having this inside `src/matcha_ml/config` and changing `GlobalConfigurationService` to `GlobalConfiguration`.

Reply:
Updated the name to `GlobalParameters`